web/src/pages/api/login-with-invite.ts

```diff
@@ -121,6 +121,6 @@ export default async function handleLogin(
     createdUser.id,
     createdUser.team_id
   );
-  setCookie("auth", { token }, req as NextApiRequest, res, expiration);
+  setCookie("auth", { token }, req as NextApiRequest, res, expiration, "lax");
```

Comment:
why is `lax` needed here? the cookie should only be used in the developer portal

Reply:
For some reason without "lax" you can't receive this cookies in GetServerSideProps after you set it.
Anyway, this is a page with invites that will be deleted in #404